lib/mpl_toolkits/tests/test_axisartist_axislines.py

```diff
@@ -83,7 +83,7 @@ def test_ParasiteAxesAuxTrans():
         ax1 = SubplotHost(fig, 1, 3, i+1)
         fig.add_subplot(ax1)
 
-        ax2 = ParasiteAxesAuxTrans(ax1, IdentityTransform())
```

Comment:
This is `test_ParasiteAxesAuxTrans`; I don't think it should be changed.

Reply:
the functionality of ParasiteAxesAuxTrans was fully folded into ParasiteAxes, so it should still be tested -- just as part of ParasiteAxes.  I guess I could rename the test to test_parasite_auxtrans?

Reply:
But `ParasiteAxesAuxTrans` has a second copy of all that code, so if it's not tested, then it could break separately from `ParasiteAxes`.

Reply:
ah, I see; parametrized the test to check both classes.
this revealed that I needed to keep the `__init__` on the AuxTransBase class too, because otherwise the deprecation machinery grabs `object.__init__` instead and wraps *that* to install the deprecation emitter.